projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```diff
@@ -29,6 +29,25 @@
 .jp-carousel-wrap.swiper-container {
 	height: auto;
 	width: 100vw;
+	opacity: 0;
+}
+
+.jp-carousel-fade-in-image {
+	animation: fadeInOpacity ease-in 0.5s 1 normal forwards;
+	-webkit-animation: fadeInOpacity ease-in 0.5s 1 normal forwards;
```

Comment:
Do we still need these vendor specific extensions? I think all the browsers we support can handle the base css3 animation properties, but could be wrong.

Reply:
Good question. At first I didn't think we needed them, but I couldn't get it to work on mobile Chrome/Safari without them. 

I'll try again. Maybe I was hallucinating. I did drink a lot of Rooibos tea today.

Reply:
Yes, we do need `webkit`, but I've ditched mozilla and the other one. Thanks for picking up on that. 👍